code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm

```diff
@@ -96,6 +96,12 @@
 	icon_state = "sec"
 	open_door_sprite = "sec_door"
 
+/obj/structure/closet/secure_closet/evidence
```

Comment:
Please make this like the cyberiad detective's locker:
Grey theme, anchored.
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/47ed20b2-d1fd-4cb5-9cbe-580b32d9ff76)

Please also make a special detective subtype with `ACCESS_FORENSICS_LOCKERS` and replace the var-edited detective closet with the new one you make.

Reply:
![Screenshot 2024-02-13 115349](https://github.com/ParadiseSS13/Paradise/assets/98861947/b187ddea-090e-493a-a09c-8adeef9a2fb6)
Like this?

Reply:
there should be 2 types, that type but with the normal sec access, and a subtype with the forensics_locker access

Reply:
nearly there, let me put a suggestion for how to finish it

Reply:
https://github.com/ParadiseSS13/Paradise/pull/24037/files#r1488356219

Reply:
Could you revert the icon_state back to the grey state, and anchor it please? Also, these 2 additions should be near each other in the files, and not in the middle of a definition for another type (in this case, between /obj/structure/closet/secure_closet/security/ and /obj/structure/closet/secure_closet/security/populate_contents()).

I'd recommend putting both right after /obj/structure/closet/secure_closet/security/populate_contents()

Reply:
fixed the issue